lib/disability_compensation/providers/rated_disabilities/lighthouse_rated_disabilities_provider.rb

```diff
@@ -12,6 +12,21 @@ def initialize(icn)
     @icn = icn
   end
 
+  # @param [string] lighthouse_client_id: the lighthouse_client_id requested from Lighthouse
+  # @param [string] lighthouse_rsa_key_path: path to the private RSA key used to create the lighthouse_client_id
+  def get_disability_rating(lighthouse_client_id, lighthouse_rsa_key_path)
+    auth_params = {
```

Comment:
it looks like we can reuse this chunk of code starting from line 18 - 25 and 33 - 40 in both `get_disability_rating` and `get_rated_disabilities` methods!
should we make a shareable method that can be called in both methods?

Reply:
good catch.